### ISSUE TITLE
Does not work with Kirby 3.8

### ISSUE THREAD

**Issue by distantnative** [rasteiner/k3-whenquery#12]
As it is easy to overlook on Discord:

> For the love of — Today at 10:31 AM
@rasteiner I was just trying out the 3.8.0 rc and noticed your amazing whenQuery plugin was no longer working. I have not been able to thoroughly test it but it seems to always return true. 
Excuse me if this is not the right place for this message

> distantnative : nico — Today at 10:44 AM
@rasteiner I think it is this line https://github.com/rasteiner/k3-whenquery/blob/main/src/index.ts#L39 where you are hacking the meetsCondition method. This got change and now there is this.$helper.field.isVisible() that you probably can use https://github.com/getkirby/kirby/blob/develop/panel/src/helpers/field.js

**Comment by rasteiner** [comment]
Thanks for the ping, the last time I've looked at discord was several weeks ago. I'm in my september - february period where I don't really have time for OS work... 

Putting [this](https://github.com/getkirby/kirby/commit/130211318359de48e665d95da8344be563e2989b) here for myself and future reference.

**Comment by distantnative** [comment]
@Bastianallgeier thinking about this, I think @rasteiner won't be able to extend the functionality anymore because you now call the helper directly from the template. So it could be harder to extend the method call.

**Comment by rasteiner** [comment]
> @bastianallgeier thinking about this, I think @rasteiner won't be able to extend the functionality anymore because you now call the helper directly from the template. So it could be harder to extend the method call.

I actually thought about simply overriding the helper function itself... Should work

Something like 
```ts
const orig = Vue.prototype.$helper.field.isVisible;
Vue.prototype.$helper.field.isVisible = function(field, values) {
        if(!orig(field, values)) return false;

        if(field.whenQuery) {
             //...
```

The magic of javascript, you can tinker with pretty much anything 😁

**Comment by bastianallgeier** [comment]
I love that everything in JS can simply be monkey-patched 🙈

**Comment by rasteiner** [comment]
fixed in 0.4.0
